SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -576,10 +588,19 @@ void USpatialActorChannel::OnCreateEntityResponse(const Worker_CreateEntityRespo
 
 	if (Op.status_code != WORKER_STATUS_CODE_SUCCESS)
 	{
-		UE_LOG(LogSpatialActorChannel, Error, TEXT("Failed to create entity for actor %s: %s"), *Actor->GetName(), UTF8_TO_TCHAR(Op.message));
-		Sender->SendCreateEntityRequest(this, GetPlayerWorkerId());
-		return;
+		if (Op.status_code == WORKER_STATUS_CODE_TIMEOUT)
+		{
+			UE_LOG(LogSpatialActorChannel, Error, TEXT("Failed to create entity for actor %s Reason: %s. Retrying..."), *Actor->GetName(), UTF8_TO_TCHAR(Op.message));
+			Sender->SendCreateEntityRequest(this);
+			return;
+		}
+		else
+		{
+			UE_LOG(LogSpatialActorChannel, Error, TEXT("Failed to create entity for actor %s Reason: %s"), *Actor->GetName(), UTF8_TO_TCHAR(Op.message));
```

Comment:
Is this a driveby fix or is it related to the overlapping boundaries work?

Reply:
Yes it is. Most likely will take it out if James's changes go in.